source/declarations.tex

```diff
@@ -479,9 +479,11 @@
 \end{example}
 
 \pnum
-The \tcode{mutable} specifier shall appear only in the declaration of
+The \keyword{mutable} specifier shall appear only in the declaration of
 a non-static data member\iref{class.mem}
 whose type is neither const-qualified nor a reference type.
+A member subobject\iref{intro.object} corresponding to a non-static
+data member declared with the \keyword{mutable} specifier is a \term{mutable subobject}.
```

Comment:
\term is bad. Use \defn and friends. I think we want \defnadj{mutable}{subobject}.